docs/source/files.rst

```diff
@@ -32,6 +32,10 @@ Upload a string or bytes
 ^^^^^^^^^^^^^^^^^^^^^^^^
 .. automethod:: cognite.client._api.files.FilesAPI.upload_bytes
 
+Upload a file larger than 5GiB in multiple parts
```

Comment:
I wouldn't specify "larger than 5GiB" - that's a limitation on the single-part upload endpoint, not a limitation here. It can and will be used for files smaller than that.

Reply:
Yes, multipart upload can be used to upload even a 1-byte file.

If a client script wants to upload file sizes of any size between 1 byte and 1000 GiB, they should use the multipart upload function, and not bother switching between the old upload and the new upload_multipart